### ISSUE TITLE
Error: 500 AttributeError("'list' object has no attribute 'lower'",)

### ISSUE THREAD

**Issue by BodingClockchian** [morpheus65535/bazarr#247]
**Describe the bug**
When Bazarr is searching for all wanted subtitles it gives the following error
"Error: 500 Internal Server Error"

**To Reproduce**
Search for all subtitles, I got this error message when searching for movie subtitles

**Expected behavior**
Search and hopefully find subtitles for the movies

**Software (please complete the following information):**
 - Bazarr: Bazarr Ver. 0.6.7 ,changelog says 6.9.1?
 - OS: Windows 7

**Additional context**
`Exception:
AttributeError("'list' object has no attribute 'lower'",)
Traceback:
Traceback (most recent call last):
  File "bazarr\../libs/bottle.py", line 862, in _handle
    return route.call(**args)
  File "bazarr\../libs/bottle.py", line 1740, in wrapper
    rv = callback(*a, **ka)
  File "bazarr\main.py", line 144, in wrapper
    return func(*a, **ka)
  File "bazarr\main.py", line 800, in wanted_search_missing_subtitles_list
    wanted_search_missing_subtitles()
  File "C:\bazarr\bazarr\get_subtitle.py", line 459, in wanted_search_missing_subtitles
    wanted_download_subtitles_movie(movie[0])
  File "C:\bazarr\bazarr\get_subtitle.py", line 422, in wanted_download_subtitles_movie
    message = download_subtitle(path_replace_movie(movie[0]), str(alpha3_from_alpha2(language)), movie[4], providers_list, providers_auth, str(movie[5]), 'movie')
  File "C:\bazarr\bazarr\get_subtitle.py", line 66, in download_subtitle
    sorted_subtitles = sorted([(s, compute_score(s, video, hearing_impaired=hi)) for s in subtitles], key=operator.itemgetter(1), reverse=True)
  File "bazarr\../libs/subliminal\score.py", line 106, in compute_score
    matches = subtitle.get_matches(video)
  File "bazarr\../libs/subliminal\providers\opensubtitles.py", line 102, in get_matches
    matches |= guess_matches(video, guessit(self.movie_release_name, {'type': 'movie'}))
  File "bazarr\../libs/subliminal\subtitle.py", line 265, in guess_matches
    if frmt.lower() == video_format.lower():
AttributeError: 'list' object has no attribute 'lower'`

**Comment by morpheus65535** [comment]
Update to 0.6.9.1. This is already fixed.

**Comment by morpheus65535** [comment]
That’s strange. You’ve installed with git clone?

**Comment by BodingClockchian** [comment]
> That’s strange. You’ve installed with git clone?

I did. This error showed up

> 07/12/2018 18:58:29|ERROR|Job "Update bazarr from source on Github (trigger: interval[6:00:00], next run at: 2018-12-08 00:58:27 CET)" raised an exception|'Traceback (most recent call last):\n  File "bazarr\\../libs/apscheduler\\executors\\base.py", line 125, in run_job\n    retval = job.func(*job.args, **job.kwargs)\n  File "C:\\bazarr\\bazarr\\check_update.py", line 32, in check_and_apply_update\n    g.fetch(\'origin\')\n  File "bazarr\\../libs/git\\cmd.py", line 548, in <lambda>\n    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)\n  File "bazarr\\../libs/git\\cmd.py", line 1014, in _call_process\n    return self.execute(call, **exec_kwargs)\n  File "bazarr\\../libs/git\\cmd.py", line 825, in execute\n    raise GitCommandError(command, status, stderr_value, stdout_value)\nGitCommandError: Cmd(\'git\') failed due to: exit code(128)\n  cmdline: git fetch origin\n  stderr: \'fatal: \'origin\' does not appear to be a git repository\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists.\''|

**Comment by morpheus65535** [comment]
If you force an update again do you still have this error message?

**Comment by BodingClockchian** [comment]
> If you force an update again do you still have this error message?

Jep, log file gives the same error message

**Comment by BodingClockchian** [comment]
> I've tested it and installed a new instance on Windows 10 using git clone. I've switch back and forth between master and dev branch but I've never faced any issue when updating.
> 
> I would suggest to stop bazarr, take a copy of the data directory, remove your bazarr directory and clone it again. Before starting, restore your copy of the data directory.
> 
> Keep me informed please.

Yeah its pretty strange. But I managed to get it working again with my old settings. Like you just said, copying the data directory and getting a fresh bazarr directory with git.

Are you aware I was still using Ver. 0.6.7 Dev branch? Before the re-install

**Comment by morpheus65535** [comment]
0.6.7.1 have been release on November 6. You probably have a problem with your update process since that day. I hope you wont run into another issue like this one.
